### ISSUE TITLE
Would like to specify HTML content for options menu

### ISSUE THREAD

**Issue by timothyarmes** [silviomoreto/bootstrap-select#793]
Hi,

Great project.

I'd like to specify HTML content for the options menu that's separate from the content that's shown in the button, while still showing the main content + sub text in the button itself.

Currently I'm working around this by only showing sub-text in the button so that I can use the content for the menu options, but that's not ideal:

![screen shot 2014-11-13 at 21 54 56](https://cloud.githubusercontent.com/assets/1092550/5036428/d16b1c3a-6b7f-11e4-8b20-b883f10fc184.PNG)

Thx.

**Comment by caseyjhol** [comment]
Set data-content on each option and set `showContent` to `false`.

**Comment by timothyarmes** [comment]
Yes, that's what I'm doing in me screenshot. The problem is that I wish to display HTML in the button, but that HTML should be *different* to that that's shown in the menu.

**Comment by hsiaolee** [comment]
HI,
I use data-content is my iOS webview but it's not working,it look like this :
![2017-02-09 5 49 01](https://cloud.githubusercontent.com/assets/6904078/22777969/0ead4276-eef0-11e6-82e3-95769843b213.png)

how can I fix it?
Thx.

**Comment by nileio** [comment]
@caseyjhol can anyone help please ? what is the correct behaviour for this ?
I use showContent=false , and still the html of the selected option is shown in the button. I want to show a different value than the option html. the option html is set by using `data-content`

**Comment by nileio** [comment]
I got this to work as expected - dont know what I was doing wrong initially sorry! I also upgraded to latest release.
